### PR TITLE
Fix bug: O3-1231: Actions menu overlays the user's next action (both …

### PR DESCRIPTION
**Pull request by nanfuka**
…desktop and tablet)

## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Related

https://user-images.githubusercontent.com/29108523/165064636-9acc4ba5-5a76-4e0d-812c-a7e42fa2177e.mp4

 Issue

### CODE REVIEW

**Comment by nanfuka** [comment]
@gracepotma kindly review

File: packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx [hunk 1]
```diff
@@ -53,6 +53,11 @@ const PatientBanner: React.FC<PatientBannerProps> = ({
       !(overFlowMenuRef?.current && overFlowMenuRef?.current.contains(event.target)) && onClick(patientUuid);
     }
   };
+  const [showDropdown, setShowDropdown] = React.useState(false);
+  const diappear = React.useCallback((event: MouseEvent) => {
```
Comment (donaldkibet) [inline packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx:57]:
```suggestion
  const closeDropdownMenu = React.useCallback((event: MouseEvent) => {
```
I believe this is a typo, naming it as `closeDropdownMenu` is more descriptive of the intent of what the function does

File: packages/esm-patient-banner-app/src/ui-components/overflow-menu.component.tsx [hunk 2]
```diff
@@ -2,11 +2,20 @@ import React, { useState, useCallback, useEffect, useRef } from 'react';
 
 interface CustomOverflowMenuComponentProps {
   menuTitle: React.ReactNode;
+  dropDown: boolean;
```
Comment (donaldkibet) [inline packages/esm-patient-banner-app/src/ui-components/overflow-menu.component.tsx:5]: you could rename this to describe what the `prop` intent is

**Review by donaldkibet — CHANGES_REQUESTED** [review]

**Review by donaldkibet — APPROVED** [review]
LGTM Thanks @nanfuka

**Pull request merged**